src/main/java/net/glowstone/block/ItemTable.java

```diff
@@ -234,7 +234,6 @@ private void registerBuiltins() {
         reg(Material.BREWING_STAND_ITEM, new ItemPlaceAs(Material.BREWING_STAND));
         reg(Material.CAULDRON_ITEM, new ItemPlaceAs(Material.CAULDRON));
         reg(Material.FLOWER_POT_ITEM, new ItemPlaceAs(Material.FLOWER_POT));
-        reg(Material.SKULL_ITEM, new ItemPlaceAs(Material.SKULL));
```

Comment:
What is "this diff"
If you're wondering why I removed it it's just been moved to the bottom of the list, I can't remember why though. I think it was because I used to have all wearables in one place.